EDU-832-PT-guia-Network-Lists (#280)
aziontech/docs: src/content/docs/pt-br/pages/guias/blacklists-ip-addresses-edge/index.mdx

---
title: Como criar listas de bloqueio de IP, ASN e geolocalização com Network Lists
description: Gerencie listas de liberação e bloqueio a partir de IPs, ASNs e geolocalizações dos usuários com Network Layer Protection e Network Lists.
namespace: docs_use_case_blacklists_ip_addresses_edge
permalink: /documentacao/produtos/guias/blacklists-enderecos-ip-edge/
---

**Network Layer Protection** é um módulo de **Edge Firewall** que possibilita a criação de **Network Lists**, para permitir (*allowlists*) ou bloquear (*blocklists*) usuários de interagir com suas edge applications na Azion.

As Network Lists podem ser baseadas em endereços de IP, ASN e geolocalização do usuário. Elas podem feitas considerando o escopo real da aplicação, ou desenvolvidos e mantidos pela Azion, como a lista de *IPs da rede Tor*.

> > Saiba mais sobre [Network Lists](/pt-br/documentacao/produtos/edge-firewall/network-layer-protection/network-lists), [Network Layer Protection](/pt-br/documentacao/produtos/edge-firewall/network-layer-protection/), [módulos do Edge Firewall](/pt-br/documentacao/produtos/edge-firewall/) e o [Rules Engine para Edge Firewall](/pt-br/documentacao/produtos/edge-firewall/rules-engine/).

Reviewer: ```suggestion
Saiba mais sobre [Network Lists](/pt-br/documentacao/produtos/edge-firewall/network-layer-protection/network-lists), [Network Layer Protection](/pt-br/documentacao/produtos/edge-firewall/network-layer-protection/), [módulos do Edge Firewall](/pt-br/documentacao/produtos/edge-firewall/) e o [Rules Engine para Edge Firewall](/pt-br/documentacao/produtos/edge-firewall/rules-engine/).
```

Para criar, gerenciar e utilizar Network Lists, siga os passos:

1. Crie um **Edge Firewall Rule Set** com o módulo **Network Layer Protection** habilitado.
2. Crie uma **Network List**.
3. Associe esta Network List ao **Rulse Engine** de um **Edge Firewall**.

> ## Crie um Edge Firewall Rule Set

Reviewer: ```suggestion
## Criar um Edge Firewall Rule Set
```

Reviewer: uso de infinitivo para títulos PT

1. Acesse o [Real-Time Manager (RTM)](https://manager.azion.com).

> 2. Abra o **Products menu**, representado pelas três linhas horizontais, e selecione **Edge Firewall**.

Reviewer: ```suggestion
2. Abra o **Products menu**, representado pelas três linhas horizontais no canto superior esquerdo, e selecione **Edge Firewall**.
```

3. Escreva o nome de identificação da **Rule Set** no campo **Edge Firewall Name**.
4. Selecione os domínios quais deseja que o firewall esteja ativo e clique no botão `>` para movê-los para o campo **Chosen Domains**.
5. Certifique-se de que o switch **Network Layer Protection** esteja habilitado na seção **Edge Firewall Modules**.
6. Certifique-se de que o switch **Active** esteja habilitado.
7. Clique no botão **Save**.

Você pode ver seu rule set criado na lista de edge firewalls.

### Ative o módulo Network Layer Protection

Para **Edge Firewall Rule Sets** já criados, siga os passos:

1. Acesse o [Real-Time Manager (RTM)](https://manager.azion.com).

> 2. Abra o **Products menu**, representado pelas três linhas horizontais, e selecione **Edge Firewall**.

Reviewer: ```suggestion
2. Abra o **Products menu**, representado pelas três linhas horizontais no canto superior esquerdo, e selecione **Edge Firewall**.
```

3. Na lista de Edge Firewall, selecione a Rule Set que você deseja adicionar a uma **Network List**.

> 4. Na aba **Main Settings do Edge Firewall**, habilite o switch **Network Layer Protection**.

Reviewer: não precisa salvar aqui?

Author: Não, pois está indicando para o usuário acessar uma configuração já criada.

> ## Crie uma Network List

Reviewer: ```suggestion
## Criar uma Network List
```

1. Acesse o [Real-Time Manager (RTM)](https://manager.azion.com).

> 2. Abra o **Products menu**, representado pelas três linhas horizontais, na seção **Edge Libraries** selecione **Network Lists**, from the **Edge Libraries** section.
>    > Ao abrir a página de Network Lists, todas as listas criadas pelo usuário e as fornecidas automaticamente pela Azion serão exibidas.
> 3. Clique no botão **Add**.
> 4. Preencha os seguintes campos:

Reviewer: ```suggestion
2. Abra o **Products menu**, representado pelas três linhas horizontais no canto superior esquerdo.
3. Na seção **Edge Libraries**, selecione **Network Lists**.
   - Ao abrir a página de Network Lists, todas as listas criadas pelo usuário e as fornecidas automaticamente pela Azion serão exibidas.
4. Clique no botão **Add**.
5. Preencha os seguintes campos:
```

| Campo | Descrição |
|--------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| `Add Network List` | Nome de identificação da sua Network List. Esse nome aparecerá na lista de opções na seção *Criteria*, na configuração do **Edge Firewall**. |
| `Tipo` | Tipo da Netrwork List: <br />  `Autonomous System Number (ASN)` <br /> `Countries` <br />  `IP/CIDR` |
| `List` | Adicione os itens que comporão sua lista aqui. |

Para os tipos de lista `ASN` e `IP/CIDR`, será exibido um campo de texto. Os itens da lista devem ser separados por linha e você deve escrever um endereço por linha. Itens duplicados serão excluídos. Para o tipo `Countries`, será apresentada uma lista de seleção de países.

> 5. Clique no botão **Save**.

Reviewer: ```suggestion
6. Clique no botão **Save**.
```

Reviewer: Sugestão:
Deixar o passo 6 de Save logo da tabela dos campos. 
Logo o parágrafo: Para os tipos de lista `ASN` e `IP/CIDR`...
Finalmente o exemplo de lista.

Exemplo de lista do tipo IP/CIDR:

```
123.456.789.1
123.456.789.2/32
10.1.1.0/16
```

> ## Associe a Network List ao Edge Firewall Rule Set

Reviewer: ```suggestion
## Associar a Network List ao Edge Firewall Rule Set
```

1. Acesse o [Real-Time Manager (RTM)](https://manager.azion.com).

> 2. Abra o **Products menu**, representado pelas três linhas horizontais, e selecione **Edge Firewall**.

Reviewer: ```suggestion
2. Abra o **Products menu**, representado pelas três linhas horizontais no canto superior esquerdo, e selecione **Edge Firewall**.
```

> 3. Selecione o **Edge Firewall Rule Set** que você criou ou configurou na [primeira seção](#crie-um-edge-firewall-rule-set).

Reviewer: ```suggestion
3. Selecione o **Edge Firewall Rule Set** que você criou ou configurou na [primeira seção](#criar-um-edge-firewall-rule-set).
```
